scripts/nats-bootstrap/functions.sh

```diff
@@ -12,7 +12,9 @@ function clean_natsserver() {
 }
 
 function init_natsserver() {
+	kubectl get pods
```

Comment:
with out a `diff` it's hard to understand why we do 2 `get pods`?

Reply:
Oh, that is mainly to just eyeball before and after the helm deployment when debugging